sonar-scala-plugin/pom.xml

```diff
@@ -24,8 +24,9 @@
 
     <properties>
         <scoverage.plugin.version>1.3.0</scoverage.plugin.version>
-        <scala.version>2.12</scala.version>
+        <scala.version>2.12.7</scala.version>
```

Comment:
What's the reason for that?

Reply:
scapegoat analyser was failing with 2.12, 2.12.2, so I updated to the latest version 2.12.7 (even if it's working with 2.12.6, I don't see a benefit to use an older version)